src/videojs-contrib-hls.js

```diff
@@ -289,6 +290,7 @@ class HlsHandler extends Component {
     if (tech.options_ && tech.options_.playerId) {
       let _player = videojs(tech.options_.playerId);
 
+      _player.gapSkipper();
```

Comment:
I don't think videojs calls this function, isn't this how you instantiate a plugin? Either way this is all subject to change because I am turning it into a module, so stay tuned.